config.yaml

```
@@ -302,81 +302,81 @@ languages:
+        - title: スポーツ分析
+          text: クリケット分析は、統計的モデリングと予測分析によって選手やチームのパフォーマンスを向上させることで、クリケットの試合を変えようとしています。NumPyはこれらの沢山の分析を実現可能にしました。
           img: /images/content_images/case_studies/sports.jpg
-          alttext: Cricket ball on green field.
+          alttext: 緑のフィールド上にボールが存在している。
           url: /ja/case-studies/cricket-analytics
-        - title: Pose Estimation using deep learning
-          text: DeepLabCut uses NumPy for accelerating scientific studies that involve observing animal behavior for better understanding of motor control, across species and timescales.
+        - title: 深層学習による姿勢推定
+          text: DeepLabCutでは、動物の行動を観察することで、動物の種類や時間軸を超えた運動制御の理解を深めることで科学技術研究を加速するために、NumPyを使用しています。
           img: /images/content_images/case_studies/deeplabcut.png
-          alttext:  Cheetah pose analysis
+          alttext:  チータの姿勢推定
           url: /ja/case-studies/deeplabcut-dnn
     
       keyfeatures:
         features:
-        - title: Powerful N-dimensional arrays
-          text:  Fast and versatile, the NumPy vectorization, indexing, and broadcasting concepts are the de-facto standards of array computing today.
-        - title: Numerical computing tools
-          text: NumPy offers comprehensive mathematical functions, random number generators, linear algebra routines, Fourier transforms, and more.
-        - title: Interoperable
-          text: NumPy supports a wide range of hardware and computing platforms, and plays well with distributed, GPU, and sparse array libraries.
-        - title: Performant
-          text: The core of NumPy is well-optimized C code. Enjoy the flexibility of Python with the speed of compiled code.
-        - title: Easy to use
-          text: NumPy's high level syntax makes it accessible and productive for programmers from any background or experience level.
-        - title: Open source
-          text: Distributed under a liberal [BSD license](https://github.com/numpy/numpy/blob/master/LICENSE.txt), NumPy is developed and maintained [publicly on GitHub](https://github.com/numpy/numpy) by a vibrant, responsive, and diverse [community](/ja/community).
+        - title: 強力な多次元配列
+          text:  NumPyの高速で多機能なベクトル化計算、インデックス処理、ブロードキャスティングのコンセプトは、今日の配列計算のデファクト・スタンダードです。
+        - title: 数値計算ツール群
+          text: NumPyは、様々な数学関数、乱数生成器、線形代数ルーチン、フーリエ変換などを提供しています。
+        - title: 相互運用性
+          text: NumPyは、幅広いハードウェアとコンピューティング・プラットフォームをサポートしており、分散処理、GPU、疎行列ライブラリにも対応しています。
+        - title: 高パフォーマンス
+          text: NumPyの中核は最適化されたC言語のコードです。Pythonの柔軟性とコンパイルされたコードのスピードを楽しむことができます。
+        - title: 使いやすさ
+          text: NumPyの高度なシンタックスは、どんなバックグラウンドや経験値のプログラマーでも利用でき、生産性高めることができます。
+        - title: オープンソース
+          text: 自由な[BSDライセンス](https://github.com/numpy/numpy/blob/master/LICENSE.txt)で公開されており, NumPyは活発で, 互いを尊重し, 多様性を認め合う[コミュニティ](/ja/community)によって、 [GitHub](https://github.com/numpy/numpy)上でオープンに開発されています.
```

「自由な」(= Free) license is not appropriate here, since the 'liberal' roughly means that this license is not as strict as GPL.
```suggestion
          text: 寛容な[BSDライセンス](https://github.com/numpy/numpy/blob/master/LICENSE.txt)で公開されています。NumPyは活発で、互いを尊重し、多様性を認め合う[コミュニティ](/ja/community)によって、 [GitHub](https://github.com/numpy/numpy)上でオープンに開発されています.
```